homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -204,7 +204,8 @@ def update(self):
         if self._destination is not None and self._origin is not None:
             try:
                 params = WazeRouteCalculator.WazeRouteCalculator(
-                    self._origin, self._destination, self._region)
+                    self._origin, self._destination, self._region,
+                    log_lvl=None)
```

Comment:
Why not `debug`?  That way people can at least turn on logging to see what's going on if they're having issues.

Reply:
Fair point. Looking at the [code of the module](https://github.com/kovacsbalu/WazeRouteCalculator/blob/e141e3423a02ca65e5f7aba1d11499e10aaf1acf/WazeRouteCalculator/WazeRouteCalculator.py#L27), I am actually thinking I should set ```warning``` instead of ```debug``` here, as the latter would let the module actually spit out more info than it does now. Can you verify if I am correct?

Reply:
The library is using `log_level` to call `setLevel` which essentially acts as a filter.... uhg...  Does this overwrite what is configured in the `configuration.yaml`?

If you think there are useful logs out of the library, you could pass the effective level from of the platform, so at least we could get the messages if desired.

Reply:
That's above my skillset, unfortunately. I see you self-assigned, thanks.

Reply:
It's setting the log level of the module logger, so we should still be able to filter out log messages of lower levels, with logger component, but we can't "filter in" messages lower than what the module logger has set. So we should set the module logger to `debug`.

Reply:
Setting to debug.